pkg/alert/commands.go

```diff
@@ -318,3 +316,187 @@ func NewDeleteAlertSettingConditionCommand() *cobra.Command {
 
 	return cmd
 }
+
+//NewGetAlertChannelsCommand creates the `alertchannels` command
+func NewGetAlertChannelsCommand() *cobra.Command {
+	var (
+		page         int
+		pageSize     int
+		sortField    string
+		sortOrder    string
+		templateName string
+		channelName  string
+		details      bool
+	)
+
+	cmd := &cobra.Command{
+		Use:              "alertchannels",
+		Short:            "Print the registered alert channels",
+		Example:          `alertchannels --page=1 --pageSize=10 --sortField="name" --sortOrder="asc" --templateName="test" --channelName="slack" --details`,
+		TraverseChildren: true,
+		SilenceErrors:    true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var (
```

Comment:
I can say that for this case I would have done it with shorthand instead of var. 

```go
if details {
  alert, err :=
  if err != nil {} etc.
return 
}
```
Same for the rest.

Reply:
Won't we end up with two checks for `err` ?

Reply:
OK corrected